snews_pt/snews_sub.py

```diff
@@ -21,37 +21,67 @@ def save_message(message, outputfolder, return_file=False):
     """
     file = make_file(outputfolder)
     with open(file, 'w') as outfile:
-        json.dump(message, outfile, indent=4, sort_keys=True)
+        try:
+            json.dump(message, outfile, indent=4, sort_keys=True)
+        except TypeError as te:
+            json.dump(message.content, outfile, indent=4, sort_keys=True)
     if return_file:
         return file
 
 def display(message):
     """ Function to format output messages
     """
     click.echo(click.style('ALERT MESSAGE'.center(65, '_'), bg='red', bold=True))
-    for k, v in message.items():
-        key_type = type(v)
-        if key_type == type(None):
-            v = 'None'
-
-        if key_type in [int, float, str]:
-            if k=='alert_type':
-                if v=='RETRACTION':
-                    click.echo(f'{k:<20s}' + click.style(f':{v:<45}', bg='red'))
-                elif v=='UPDATE':
-                    click.echo(f'{k:<20s}' + click.style(f':{v:<45}', bg='blue'))
+    try:
```

Comment:
here instead of repeating the block, isn't it easier to check something like;
```python
if isinstance(message, JSONBlob):
    message = message.content
```

Reply:
I knew there was a better way.  I have removed the try/except blocks and checked with isstance()